docs/source/60_building_apps.md

```diff
@@ -77,7 +83,7 @@ Add the new command to the default list of commands in `ADAMMain`.
 
 Build ADAM and run the new command via `adam-submit`.
 
-```bash
+```
```

Comment:
Ah, I see. Thanks for the clarification; I thought it was an incidental deletion, but you are right, this should not be `bash` formatted.